db/seeds.rb

```diff
@@ -238,8 +238,8 @@
 broadcast_messages = {
   set_up_profile: "Welcome to DEV! 👋 I'm Sloan, the community mascot and I'm here to help get you started. Let's begin by <a href='/settings'>setting up your profile</a>!",
   welcome_thread: "Sloan here again! 👋 DEV is a friendly community. Why not introduce yourself by leaving a comment in <a href='/welcome'>the welcome thread</a>!",
-  twitter_connect: "You're on a roll! 🎉 Let's connect your <a href='/settings'> Twitter account</a> to complete your identity so that we don't think you're a robot. 🤖",
-  github_connect: "You're on a roll! 🎉 Let's connect your <a href='/settings'> GitHub account</a> to complete your identity so that we don't think you're a robot. 🤖",
+  twitter_connect: "You’re on a roll! 🎉 Do you have a Twitter account? Consider <a href=‘/settings’>connecting it</a> so we can @mention you if we share your post via our Twitter account <a href='https://twitter.com/thePracticalDev'>@thePracticalDev</a>.",
```

Comment:
I think we need to fix the quotes in the `<a>` tag around the settings link from `‘` to `'`

Reply:
That's so strange... just made the change!

Reply:
Oh dang, thanks for catching that @rhymes! I agree with you. I missed that when reviewing on mobile.

Reply:
@juliannatetreault sometimes if you copy-paste from other sites, the quotation marks end up actually being different characters. So that might be the explanation here. Generally you probably want to actually type out the ‘ character so that you accidentally don’t get a different one!

Reply:
@vaidehijoshi That's exactly what happened... copy-pasta gone wrong. 🤦‍♀️